specs-go/go.mod

```diff
@@ -1,3 +1,3 @@
 module github.com/opencontainers/distribution-spec/specs-go
 
-go 1.15
+go 1.16
```

Comment:
there was a request in zoom chat, i don't remember by whom, to keep this at the lower value because..

As per: https://golang.org/doc/modules/gomod-ref#go
the version of go in go.mod is to be the minimum.